cocos/ui/UIScrollView.cpp

```diff
@@ -1342,8 +1342,8 @@ void ScrollView::copySpecialProperties(Widget *widget)
     if (scrollView)
     {
         Layout::copySpecialProperties(widget);
-        _innerContainer = scrollView->_innerContainer;
         setDirection(scrollView->_direction);
+        setInnerContainerPosition(scrollView->getInnerContainerPosition());
```

Comment:
@pandamicro That's a much more complete solution indeed. If @zilongshanren is cool with it too I'll update the pull request.